### ISSUE TITLE
New Issue -Wheres Yigal?

### ISSUE THREAD

**Issue by omerkess** [AmirSimantov/pandit#3]
@AmirSimantov  - Is Yigal part of you PANDIT GitHub team? and if not - Yigal please join us!
If So - is there a way to see the team? I would like to assign him with a lovely task. 
This is Yigal:
![picture-208-1385751389 1](https://cloud.githubusercontent.com/assets/22465910/19228502/95606064-8ece-11e6-9dc7-e2e84094bc3d.png)

**Comment by ybronner** [comment]
Yigal seems to be a contributor

**Comment by AmirSimantov** [comment]
@omerkess - The "team" is displayed only once we create - what is called here - "Organization". Creating such needs paying, I think.
